config/locales/views/cookie_statement.en.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
Again isolating all of these locals that are specific to a single static view is a good step foward. There are quite a lot keys here so grouping them all under a page would help me feel confident in making a change without having to hunt through whatever else in the big en.yml might be related 👍